packages/bodiless-documentation/doc/Development/Guides/DesignElementConcepts.md

```diff
@@ -1166,24 +1150,23 @@ export const query = graphql`
 `;
 ```
 
-Feel free to delete `sites/minimal-demo/src/components/Page/index.tsx` that came
-with new site as it's no longer being used and now use our package level Page.
+Feel free to delete the `/sites/mysite/src/components/Page/index.tsx` file that came with your new
+site, as it's no longer being used and we're now using our package-level `Page`.
 
-### Try the new template
+### 6e. Try the New Template
 
-In edit mode, click on Page | New and create a new page and go to that new page.
-This page should behave identical look & behavior to the previous hard coded
-gallery's index.tsx file.
+In edit mode, click on **Page > New**, create a new page, and then go to that new page. This page
+should have the identical look and behavior of the previous hard-coded gallery's `index.tsx` file.
 
-If you look at the `src/data/pages/YOURPAGENAME`, you will see a single file
-index.json that its contents `{ "#template": "_default" }` say its using the
-site's_default tempalte.  A standard gatsby template convention.
+If you look in the `src/data/pages/YOURPAGENAME` directory, you will see a single file
+(`index.json`), and its contents (`{ "#template": "_default" }`) will say it's using the site's
+`_default` template — a standard Gatsby template convention.
 
 Feel free to add components and text to your new page.
 
-At this point, your entire site is running from code within the package and no
-code is within the page. This is site of your design system package that is easy
-to publish, reuse, extend in future.
+At this point, your entire site is running from code within the package and no code is within the
+page. This is site of your design system package that is easy to publish, reuse, and extend in the
+future.
```

Comment:
Regarding—
> This is site of your design system package that is easy to publish, reuse, and extend in the
future.

—I attempted to fill in the blanks of the beginning of this sentence, but wasn't sure and didn't want to mess up the intended message. So, this just needs an edit.